fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -2546,7 +2578,8 @@ ACTOR Future<Void> dataDistributionQueue(Reference<IDDTxnProcessor> db,
 					    .detail("AverageShardSize", req.getFuture().isReady() ? req.getFuture().get() : -1)
 					    .detail("UnhealthyRelocations", self.unhealthyRelocations)
 					    .detail("HighestPriority", highestPriorityRelocation)
-					    .detail("BytesWritten", self.bytesWritten)
+					    .detail("BytesWritten", self.moveAverage.getTotal())
+					    .detail("BytesAverage", self.moveAverage.getAverage())
```

Comment:
The name should probably include `rate` so that the units are clear; suggestion: `AverageBytesMoveRate`

Reply:
Changed it to `BytesWrittenAverageRate`